Makefile.pre.in

```diff
@@ -1994,6 +1994,21 @@ testuniversal: all
 	$(RUNSHARED) /usr/libexec/oah/translate \
 		./$(BUILDPYTHON) -E -m test -j 0 -u all $(TESTOPTS)
 
+# Run the test suite on the iOS simulator.
+# Must be run on a macOS machine with a full Xcode install that has an iPhone SE
+# (3rd edition) simulator available, after running `make install` on a configuration
+# with --enable-framework="./iOS/testbed/Python.xcframework/ios-arm64_x86_64-simulator"
+XCRESULT=$(srcdir)/build/$(MULTIARCH).$(shell date +%s).xcresult
+.PHONY: testiOS
+testiOS:
```

Comment:
```suggestion
.PHONY: test-iOS
test-iOS:
```

?

Reply:
My thinking here was to be consistent with `testuniversal`, `cleantest`, `buildbottest` et al. However, I'm happy to change this name if there's a general preference for `test-iOS` over `testiOS`.

Reply:
All existing makefile targets are lower-case, even acronyms like `multissltest`.

Reply:
I guess that's true... although I suspect I'd twitch every time I need to spell it with capitalization that breaks the branding guide 😀

Reply:
I've updated this to `testios`